src/remote_adapter/remote_adapter_client/qt_impl/sdl_remote_adapter_qt_client.cc

```diff
@@ -0,0 +1,86 @@
+#include <iostream>
+
+#include "qt_impl/sdl_remote_adapter_qt_client.h"
+#include "qt_impl/sdl_remote_adapter_receive_thread.h"
+#include "common/constants.h"
+
+namespace lua_lib {
+
+SDLRemoteTestAdapterQtClient::SDLRemoteTestAdapterQtClient(
+            const std::string& host,
+            uint32_t port,
+            MqParams& in_params,
+            MqParams& out_params,
+            QObject* parent)
+        : QObject(parent),
+          host_(host),
+          port_(port),
+          in_mq_params_(in_params),
+          out_mq_params_(out_params)  { }
+
+SDLRemoteTestAdapterQtClient::~SDLRemoteTestAdapterQtClient() {
+    if (remote_adapter_client_ptr_) {
+        remote_adapter_client_ptr_->clear();
+    }
+}
+
+void SDLRemoteTestAdapterQtClient::connectMq() {
+    if (!isconnected_) {
```

Comment:
@aderiabin I would invert this if to avoid deep nested blocks

Reply:
@LuxoftAKutsan Applied (see fd8a7a5 )